src/views/Login.vue

```diff
@@ -107,6 +111,7 @@
             "La requête s'est terminée de façon inattendue, impossible de joindre le serveur distant."
           throw e
         }
+        this.$pStore.set("loginEmail", this.email)
```

Comment:
Now that I double-checked, you should not remember the email in case of failure, only in case of login success.

Reply:
in this case, this line is not going to be executed if any error occur in the catch section, but yes it makes sens to put it in the try section.

Reply:
Oh you're right. Misread the code.